githubdelivery/pom.xml

```diff
@@ -0,0 +1,312 @@
+<!--
+  ~    Copyright 2023 OICR
+  ~
+  ~    Licensed under the Apache License, Version 2.0 (the "License");
+  ~    you may not use this file except in compliance with the License.
+  ~    You may obtain a copy of the License at
+  ~
+  ~        http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~    Unless required by applicable law or agreed to in writing, software
+  ~    distributed under the License is distributed on an "AS IS" BASIS,
+  ~    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~    See the License for the specific language governing permissions and
+  ~    limitations under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>dockstore-support</artifactId>
+        <groupId>io.dockstore</groupId>
+        <version>${revision}${changelist}</version>
+    </parent>
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>githubdelivery</artifactId>
+    <version>${revision}${changelist}</version>
+    <packaging>jar</packaging>
+
+    <name>githubdelivery</name>
+    <description>A service for rerunning github release events</description>
```

Comment:
maybe the dockstore-deploy bits would be a service, this is more a command-line client